src/actions/integrations.ts

```diff
@@ -381,3 +381,25 @@ export function submitInteractiveDialog(submission: DialogSubmission): ActionFun
         return {data};
     };
 }
+
+export function submitEmbeddedForm(submission: DialogSubmission, postID: string, appID: string): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState();
+        submission.channel_id = getCurrentChannelId(state);
+        submission.team_id = getCurrentTeamId(state);
+        submission.submission.mm_postID = postID;
```

Comment:
This seems weird to mix camelCase and snake_case. Also, why is the `mm_` prefix needed?

Reply:
To make sure it does not collide with any other possible value that is using the same namespace.
Will fix the weird casing.